opentelemetry-exporters-newrelic/src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicExporters.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.newrelic.telemetry.opentelemetry.export;
+
+import static java.util.Collections.singleton;
+
+import com.newrelic.telemetry.Attributes;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+public class NewRelicExporters {
+
+  private static IntervalMetricReader intervalMetricReader;
+
+  /** Start up the New Relic Metric and Span exporters with the provided configuration. */
+  public static void start(Configuration configuration) {
```

Comment:
What do you think about taking it even one step further and overloading this and providing
```
public static void start(String apiKey, String appName){
    return start(new NewRelicExporters.Configuration(apiKey, "My Service Name"))
}
```
with some kind of "most common" default config?

Reply:
I think that sounds reasonable. I'll add it.

Reply:
done